postgresql/benchmark.sh

```diff
@@ -1,35 +1,51 @@
 #!/bin/bash
 
+set -eux
+
+PGVERSION=17
+
 sudo apt-get update
-sudo apt-get install -y postgresql-common
-sudo apt-get install -y postgresql-14
+sudo apt install -y postgresql-common
+sudo /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh
```

Comment:
Could you please add "https://wiki.postgresql.org/wiki/Apt" as a comment behind l. 9?

Reply:
Done